RealEstateAgencyApp.Domain/Entities/Request.cs

```diff
@@ -0,0 +1,39 @@
+﻿using RealEstateAgencyApp.Domain.Entities.Enums;
+
+namespace RealEstateAgencyApp.Domain.Entities;
+
+/// <summary>
+/// Represents a request for buying or selling real estate.
+/// </summary>
+public class Request
+{
+    /// <summary>
+    /// Gets or sets the unique identifier of the request.
+    /// </summary>
+    public required int Id { get; set; }
+
+    /// <summary>
+    /// Gets or sets the counterparty making the request.
+    /// </summary>
+    public required int CounterpartyID { get; set; }
```

Comment:
Тоже нарушение именования. Акронимы также лучше сокращать в PascalCase.
```suggestion
    public required int CounterpartyId { get; set; }
```